### DISCUSSION TITLE
(UX) add simple regex capabilities to "search on site".

### DISCUSSION THREAD

**Discussion by gety9** [plateaukao/einkbro#431] [Ideas]
Title, even one operator `OR` / `|` will make life so much easier.

Use case i want to search page for

"RAM|killing|freezing"

**Comment by plateaukao** [comment]
The "search on site" feature is provided by system WebView. If it does not support regular expression, then there's high probability that it won't be implemented.

**Comment by gety9** [comment]
closing, not worth spending time on (very infrequent use case)
